commit e6e233733355fed428e509920c5883949f848637
Author: Alexander Usyskin <alexander.usyskin@intel.com>
Date:   Wed Feb 8 00:41:45 2017 +0200

    mei: remove support for broken parallel read
    
    commit cb97fbbcac15982406e0c74cd5512a8b6fcf10b3 upstream.
    
    Parallel reads from multiple threads on a file descriptor
    are not well defined and racy. It is safer to return to original
    behavior and simply fail the additional read.
    The solution is to remove request for next read credit.
    
    Fixes: ff1586a7ea57 ("mei: enqueue consecutive reads")
    Signed-off-by: Alexander Usyskin <alexander.usyskin@intel.com>
    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1285_e6e233733355fed428e509920c5883949f848637.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1285_e6e233733355fed428e509920c5883949f848637.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
